week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,57 @@
+// Exercise 1//
```

Comment:
Meaningful names for parameters and variables - I know it's just an exercise, but it's god to remember! :)